nntrainer/app_context.cpp

```diff
@@ -29,6 +29,7 @@
 
 #include <adam.h>
 #include <sgd.h>
+#include <adamw.h>
```

Comment:
In our project, the option "SortIncludes" is enabled in the ".clang-format" file.
Therefore, the order of headers must be sorted unconditionally.

```suggestion
#include <adam.h>
#include <adamw.h>
#include <sgd.h>
```

Reply:
Fixed, with clang-format.